src/configurations/adknowledgeportal/synapseConfigs/studies.ts

```diff
@@ -137,43 +174,36 @@ export const studiesDetailsPageProps: DetailsPageProps = {
         unitDescription: 'Files',
         title: 'Metadata Files',
       },
+      tabIndex: 1,
+      className: 'metadata-table'
     },
     {
-      name: 'StandaloneQueryWrapper',
-      title: 'Study Data',
-      columnName: 'Study',
-      resolveSynId: {
-        value: true,
-      },
-      tableSqlKeys: ['study'],
+      name: "QueryWrapperPlotNav",
+      tabIndex: 1,
       props: {
-        sql:
-          "SELECT `resourceType`,`dataType`, `assay`, COUNT(`id`) AS `Files` FROM syn11346063 WHERE  (`dataSubtype` <> 'metadata' OR `dataSubtype` IS NULL) GROUP BY 1, 2,3 ORDER BY 4 DESC",
-        facetAliases,
         rgbIndex,
-        title: 'Data Files',
+        name: 'Study Data',
+        visibleColumnCount: 10,
+        tableConfiguration: {
+          showAccessColumn: true,
+          columnLinks: [
+            {
+              matchColumnName: 'study',
+              isMarkdown: false,
+              baseURL: 'Explore/Studies/DetailsPage',
+              URLColumnName: 'Study_Name',
+              wrapValueWithParens: true,
+            },
+          ],
+        },
+        sql: dataSql,
+        shouldDeepLink: false,
       },
-    },
-    {
-      name: 'Markdown',
-      // https://www.synapse.org/#!Synapse:syn12666371/wiki/595381
-      title: 'Data Updates',
-      standalone: true,
-      props: {
-        ownerId: 'syn12666371',
-        wikiId: '595381',
-      },
-    },
-    {
-      name: 'CardContainerLogic',
-      columnName: 'Related_Studies',
-      title: 'Related Studies',
-      tableSqlKeys: ['Study'],
-      props: {
-        sqlOperator: '=',
-        sql: studiesSql,
-        ...studyCardConfiguration,
+      resolveSynId: {
+        value: true,
       },
+      tableSqlKeys: ['study'],
```

Comment:
What is this tableSqlKeys for? Can I remove it?

Reply:
I think this tells the details page what key value should be injected, working hand in hand with resolveSynId.  So on this study page, the url params have study=syn5550404 for example, but this component will get the resolved value (the name of syn5550404 which is 'MSBB') which will be used to filter.

Reply:
Nevermind, I just found out that it needs to be there.

Reply:
Thank you, that makes sense!